realis.go

```diff
@@ -82,6 +84,8 @@ type RealisConfig struct {
 	transport                   thrift.TTransport
 	protoFactory                thrift.TProtocolFactory
 	logger                      Logger
+	Insecure                    bool
```

Comment:
Since this is a bool, might get confusing if we call this Insecure. Maybe call it SecureConnection.

Reply:
lets call it insecure. it's common term in https

Reply:
If other libraries do it, then I'm fine with following the status quo